MSK: Pod Identity for Openshift Rosa

### Issue submitter TODO list

- [X] I've looked up my issue in [FAQ](https://ui.docs.kafbat.io/faq/common-problems)
- [X] I've searched for an already existing issues [here](https://github.com/kafbat/kafka-ui/issues)
- [X] I've tried running `main`-labeled docker image and the issue still persists there
- [X] I'm running a supported version of the application which is listed [here](https://github.com/kafbat/kafka-ui/blob/main/.github/SECURITY.md)

### Describe the bug (actual behavior)

Hi Team,
AWS has released a new fix to allow POD Identity. Can you please implement this and release the new version for this ?

This is the dependency that needs to be changed:-
msk aws-msk-iam-auth v2.1.0 instead of v1.1.7

### Expected behavior

Application can connect AWS MSK when deployed in a POD (Openshift Rosa)The application

### Your installation details

We deploy this using helm chart in our Openshift Roda env. This will run from POD and try to make the connection with MSK.

### Steps to reproduce

Get the latest version
create a helm chart
deploy it in Openshift Rosa env (in POD)
check the log, you will see the error "Unable to load AWS Credentials from any provider chain"

### Screenshots

Na

### Logs

NA

### Additional context

NA

Also, I was trying to build this in my local system but getting this error :- 

`[ERROR] Failed to execute goal on project kafka-ui-api: Could not resolve dependencies for project com.provectus:kafka-ui-api:jar:0.0.1-SNAPSHOT: Could not find artifact com.provectus:kafka-ui-contract:jar:0.0.1-SNAPSHOT in confluent (https://packages.confluent.io/maven/) -> [Help 1]`

how to resolve this issue ?

--- a/api/pom.xml
+++ b/api/pom.xml
@@ -91,7 +91,7 @@
         <dependency>
             <groupId>software.amazon.msk</groupId>
             <artifactId>aws-msk-iam-auth</artifactId>
-            <version>1.1.7</version>
+            <version>2.1.0</version>
         </dependency>
 
         <dependency>

